src/curated-corpus/components/SplitButton/SplitButton.tsx

```diff
@@ -83,20 +83,23 @@ export const SplitButton: React.FC<SplitButtonProps> = (props) => {
     setOpen(false);
   };
 
+  //TODO @Herraj fix ButtonGroup divider color and hover bgColor
+
   return (
     <>
       <ButtonGroup
         variant="text"
-        color="default"
         ref={anchorRef}
         aria-label="split button"
+        sx={{ color: 'rgb(0, 0, 0, 0.87)' }}
       >
```

Comment:
Could this be consistent with the approaches to specify colours elsewhere? Ideally we'd limit it to just the palette and the gradient colours MUI generates for us out of primary/secondary colours. Perhaps `curationPalette.pocketBlack` would work here?